app/ui-react/packages/ui/src/Settings/OAuth/OAuthAppExpanderBody.tsx

```diff
@@ -15,52 +14,59 @@ export interface IOAuthAppExpanderBodyProps {
   i18nAlertDetail: string;
 }
 
-export class OAuthAppExpanderBody extends React.Component<
+export const OAuthAppExpanderBody: React.FC<
   IOAuthAppExpanderBodyProps
-> {
-  constructor(props: IOAuthAppExpanderBodyProps) {
-    super(props);
-  }
-  public render() {
-    return (
-      <>
-        {this.props.showSuccess && (
-          <Row>
-            <Col xs={11}>
-              <Alert type={'success'}>
-                <strong>{this.props.i18nAlertTitle}</strong>&nbsp;
-                {this.props.i18nAlertDetail}
-              </Alert>
-            </Col>
-          </Row>
-        )}
-        <Row>
-          <Col xs={12} md={8}>
-            <Form isHorizontal={true}>{this.props.children}</Form>
-          </Col>
-        </Row>
-        <Row>
-          <Col xs={12} md={8}>
-            <>
-              <Button
-                data-testid={'o-auth-app-expander-body-save-button'}
-                bsStyle="primary"
-                onClick={this.props.onSave}
-                disabled={this.props.disableSave}
-              >
-                {this.props.i18nSaveButtonText}
-              </Button>{' '}
-              <Button
-                data-testid={'o-auth-app-expander-body-remove-button'}
-                onClick={this.props.onRemove}
-                disabled={this.props.disableRemove}
-              >
-                {this.props.i18nRemoveButtonText}
-              </Button>
-            </>
-          </Col>
-        </Row>
-      </>
-    );
-  }
-}
+> = (
+  {
+    children,
+    disableRemove,
+    disableSave,
+    i18nAlertDetail,
+    i18nAlertTitle,
+    i18nRemoveButtonText,
+    i18nSaveButtonText,
+    onRemove,
+    onSave,
+    showSuccess
+  }) => {
+  return (
+    <>
+      {showSuccess && (
+        <Grid sm={11}>
+          <GridItem>
```

Comment:
This probably could also be stuck in the same grid as the other stuff, or maybe a Stack and then showSuccess could hide/show the StackItem that holds this alert?